opennms-model/src/main/java/org/opennms/netmgt/model/OnmsIpInterface.java

```diff
@@ -696,10 +696,23 @@ public void mergeMonitoredServices(OnmsIpInterface scannedIface, EventForwarder
         for (OnmsMonitoredService svc : newServices) {
             svc.setIpInterface(this);
             getMonitoredServices().add(svc);
+            svc.addMetaData("system-explicit", "explicitlyAdded", "true");
```

Comment:
How are you sure this is only called for services that are explicitly added.  I think it is possible that any new detected service would appear here.